godtools.xcodeproj/project.pbxproj.orig

```diff
@@ -47,14 +46,12 @@
 		450E44552756B6E000D4311E /* MenuItemViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 450E44522756B6E000D4311E /* MenuItemViewModel.swift */; };
 		450E44572756B6FC00D4311E /* MenuItemViewModelType.swift in Sources */ = {isa = PBXBuildFile; fileRef = 450E44562756B6FC00D4311E /* MenuItemViewModelType.swift */; };
 		450EF834247EB3D300978926 /* SF-Pro-Text-Bold.otf in Resources */ = {isa = PBXBuildFile; fileRef = 450EF833247EB3D300978926 /* SF-Pro-Text-Bold.otf */; };
-		450F94CB27A1EEF7009DEA43 /* CardCollectionPageCard+MobileContentRenderer.swift in Sources */ = {isa = PBXBuildFile; fileRef = 450F94CA27A1EEF7009DEA43 /* CardCollectionPageCard+MobileContentRenderer.swift */; };
+		450F94CB27A1EEF7009DEA43 /* CardCollectionPageCard+MobileContentRenderableModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 450F94CA27A1EEF7009DEA43 /* CardCollectionPageCard+MobileContentRenderableModel.swift */; };
```

Comment:
I don't think think this file should actually be tracked in git, it is produced when there is a merge conflict on `project.pbxproj`. in fact it could be beneficial to define a `.gitignore` rule to ignore `*.orig`

Reply:
Good catch.  Just removed that.  Yeah I think that's a good idea to add that to the .gitignore.